app/views/provider_interface/conditions/edit.html.erb

```diff
@@ -0,0 +1,41 @@
+<% content_for :browser_title, 'Confirm conditions' %>
+<% content_for :before_content, govuk_back_link_to(provider_interface_application_choice_path(@application_choice.id)) %>
+
+<%= form_with model: @conditions_form,
+      url: provider_interface_application_choice_confirm_update_conditions_path(@application_choice.id),
+      method: :patch do |f| %>
+  <%= f.govuk_error_summary %>
+
+  <h1 class="govuk-heading-xl">
+    Confirm if the candidate has met all conditions
+  </h1>
+
+  <%= render SummaryListComponent, rows: [
+    { key: 'Candidate name', value: @application_choice.application_form.full_name },
+    { key: 'Course', value: @application_choice.course.name_and_code },
+    { key: 'Preferred location', value: @application_choice.site.name },
+    { key: 'Provider', value: @application_choice.course.provider.name_and_code },
+  ] %>
```

Comment:
Perhaps we can extract https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/app/views/provider_interface/decisions/new_offer.html.erb#L16-L22 into a component to use here?

Reply:
Ok, I'll create a `ConditionsComponent`

Reply:
I've created a component, however it doesn't look like we can have a single component for all cases. I've opened a draft card here: https://trello.com/c/JJZ1yaTg